src/descriptor/dsl.rs

```diff
@@ -317,10 +354,10 @@ macro_rules! apply_modifier {
 ///
 /// let b_items = vec![
 ///     bdk::fragment!(pk(my_key_1))?,
-///     bdk::fragment!(s:pk(my_key_2))?,
-///     bdk::fragment!(s:d:v:older(my_timelock))?,
+///     bdk::fragment!(s: pk(my_key_2))?,
+///     bdk::fragment!(s: d: v: older(my_timelock))?,
```

Comment:
spaces inserted on modifier by rustfmt doesn't look good, maybe adding a `#[rustfmt(skip)]` is slightly less ugly

same on `policy.rs:1154`

Reply:
Yeah, makes sense